### ISSUE TITLE
Searching for apex process... Init with qemu connector... Can't create qemu connector Init with kvm connector... Can't create kvm connector Quitting

### ISSUE THREAD

**Issue by qingchan3366** [MisterY52/apex_dma_kvm_pub#37]
Searching for apex process...
Init with qemu connector...
Can't create qemu connector
Init with kvm connector...
Can't create kvm connector
Quitting

**Comment by albatror** [comment]
Look on build folder if you see the 3 lib (.so) for qemu, kvm and win32, if you dont see them, just put them manually in the build folder then restart.

**Comment by pppaluarit** [comment]
@albatror can i use your fork in single gpu passthrough?
what are you using for obfuscate Client.exe and Overlay.exe file?
thanks

**Comment by albatror** [comment]
yes you can use it, it not depending about 1 or 2 gpu and to obfuscate files yo can use THEMIDA(64) or VMprotect for example or any soft like this

**Comment by pppaluarit** [comment]
thank you so much, ill try it.

**Comment by Harvey518** [comment]
> Look on build folder if you see the 3 lib (.so) for qemu, kvm and win32, if you dont see them, just put them manually in the build folder then restart.
hey,i put the 3lib(.so) in the build folder and it still show me this
Searching for apex process...
Init with qemu connector...
Can't create qemu connector
Init with kvm connector...
Can't create kvm connector
Quitting
How can i solve this

**Comment by Harvey518** [comment]
@albatror 
hey,i put the 3lib(.so) in the build folder and it still show me this
Searching for apex process...
Init with qemu connector...
Can't create qemu connector
Init with kvm connector...
Can't create kvm connector
Quitting
How can i solve this?

**Comment by albatror** [comment]
hi, have you tried : 
`curl --proto '=https' --tlsv1.2 -sSf https://sh.memflow.io | sh`

**Comment by Harvey518** [comment]
> hi, have you tried : `curl --proto '=https' --tlsv1.2 -sSf https://sh.memflow.io | sh`

already try this.

**Comment by albatror** [comment]
I apologize for my mistake dude (i'm fucking dumb), i just reinstall everything to test it from nothing on my new computer and 
 just saw the diff between my local repo and the one on github.
I will update my repo to fix this soon (working on PDB/CR3 actually).
I invite you to :
Delete memflow_lib folder in adkv/apex_dma
Donwload repo from MisterY52, copy memflow_lib folder to adkv/apex_dma
Run build.sh and normally it will be good.
Tell me if it work for you.

**Comment by Feilink1** [comment]
Hi @albatror, I am having the same issue, I followed the instructions in your Readme section and copied MisterY52 memflow_lib folder to adkv/apex_dma, ran the install_memflow script then built apex_dma. It keeps giving me

```
Searching for apex process...
Init with qemu connector...
Can't create qemu connector
Init with kvm connector...
Can't create kvm connector
Quitting
``` 

Btw I had to change in the build.sh script `memflow_ffi_win32` to `memflow_ffi` to make the build work. I need your help, thank you in advance. Kind regards

**Comment by albatror** [comment]
Have you followed all steps ?
```
Download sources from there *.zip or git
Extract it
Install Cargo & Rust with the famous curl https://sh.rustup.rs/ -sSf | sh
Install memflow with the famous curl --proto '=https' --tlsv1.2 -sSf https://sh.memflow.io/ | sh
Compile with the build.sh to see if any errors, if errors, fix it/them then redo build.
When build is ok without errors, Download from https://github.com/albatror/memflow/blob/main/install_memflow.sh
Make the script executable, Edit it, Read and Save.
Run it. You will see if its OK in the console :) then you will see in /apex_dma/memflow_lib/ 2 new folders :
memflow-kvm
memflow-qemu
Go back to rebuild the complete sources.
```
hope it will help.

**Comment by makawy7** [comment]
Hi @albatror, I use your fork and I manged to build host successully, but should I compile apex_guest on the vm, or can i use another windows pc to compile?
because when I run sudo ./apex_dma, it keep saying "can't find client process" even after adding the offset the client.exe gives me on the windows vm inside apex_dma.cpp?!
although it can detect apex process and it keep doing the dtb shuffling.
